tests/containers/run_container_in_k3s.pm

```diff
@@ -30,11 +30,30 @@ sub run {
 
     assert_script_run("curl -O " . data_url("containers/k8s_job_manifest.yaml"));
     file_content_replace("k8s_job_manifest.yaml", JOB_NAME => $job_name, IMAGE => $image, CMD => $cmd);
-    assert_script_run("kubectl apply -f k8s_job_manifest.yaml", timeout => 300);
+    assert_script_run("kubectl apply -f k8s_job_manifest.yaml", timeout => 600);
     wait_for_k8s_job_complete($job_name);
     my $pod = find_pods("job-name=$job_name");
     validate_pod_log($pod, "SUSE Linux Enterprise Server");
     record_info('cmd', "Command `$cmd` successfully executed in the image.");
 }
 
+sub cleanup {
+    my ($self) = @_;
+    uninstall_k3s();
+}
+
+sub post_fail_hook {
+    my ($self) = @_;
+    script_run('kubectl get nodes');
```

Comment:
If this is debug output, I would wrap this into a `record_info`

Reply:
done